python/pyspark/sql/functions.py

```diff
@@ -1675,15 +1675,18 @@ def array(*cols):
 @since(1.5)
 def array_contains(col, value):
     """
-    Collection function: returns True if the array contains the given value. The collection
-    elements and value must be of the same type.
+    Collection function: returns null if the array is null, true if the array contains the
+    given value, and false otherwise.
 
     :param col: name of column containing array
     :param value: value to check for in array
 
     >>> df = spark.createDataFrame([(["a", "b", "c"],), ([],)], ['data'])
     >>> df.select(array_contains(df.data, "a")).collect()
     [Row(array_contains(data, a)=True), Row(array_contains(data, a)=False)]
+    >>> df = spark.createDataFrame([(["1", "2", "3"],), ([],)], ['data'])
+    >>> df.select(array_contains(df.data, 1)).collect()
```

Comment:
generally docstring we keep as simple as possible - any particular reason to add "1" vs the existing "a"?